liquid-rust-lrir/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(rustc_private)]
+#![feature(const_panic)]
```

Comment:
Why do we need this?

Reply:
Because the `newtype_index` macro panics if you try to create an index that doesn't fit inside an `u32`. I don't love it either.